recipe/meta.yaml

```diff
@@ -36,4 +36,5 @@ about:
 
 extra:
   recipe-maintainers:
+    - bgruening
```

Comment:
Are you ok being a maintainer here, @bgruening?

Reply:
Yes, this is ok. Thanks for asking.